metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/hook/UpdateIndicesHook.java

```diff
@@ -231,7 +231,8 @@ private void updateFineGrainedEdgesAndRelationships(
         // for every downstream, create an edge with each of the upstreams
         for (Urn downstream : fineGrainedLineage.getDownstreams()) {
           for (Urn upstream : fineGrainedLineage.getUpstreams()) {
-            edgesToAdd.add(new Edge(downstream, upstream, DOWNSTREAM_OF));
+            // TODO: add edges uniformly across aspects
```

Comment:
What does this todo mean exactly?

Reply:
yeah it's just that we're adding edges differently for InputFields and fineGrainedLineages right now (like this method and the one below it) and once we add some changes to specifying what the source is, we should be able to remove these methods and just create edges properly along with all the other edges

Reply:
Okay cool